web/components/application/hacker/pages/PageTwo.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import update from 'immutability-helper';
+import PropTypes from 'prop-types';
+
+import propTypes from '../../../../prop-types';
+import { TextInput } from '../../../input/text';
+
+const PageTwo = (props) => {
```

Comment:
Just copy pasted page one's design because i'm not worried about design yet

Reply:
a brief comment noting this might be good!